packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -1382,6 +1381,16 @@ class UiKitViewController {
   }
 }
 
+/// Controls an iOS UIView.
+///
+/// Typically created with [PlatformViewsService.initUiKitView].
+class UiKitViewController extends DarwinPlatformViewController {
```

Comment:
This seemed useful in case we need to make decoupling changes between iOS and macOS, especially as the macOS implementation is still planned.

Reply:
We need it for backward compatibility since these classes have existed for a long time on iOS. Longer term, it may well turn out we can merge all of this into a single class, but that'll require a deprecation process.

Reply:
I'm wondering if there is a `typealias` in dart? 

`typealias UiKitViewController = DarwinPlatformViewController`